torch/ao/ns/fx/graph_passes.py

```diff
@@ -396,14 +426,40 @@ def _can_insert(node_a_arg, gm_a):
 
     # For each node, check if we handle the copy behavior. This follows the
     # logic in `_insert_copy_of_subgraph_a_after_input_node_c`.
-    for node_a_arg in nodes[0].args[num_non_param_args_node_a:]:
-        if not _can_insert(node_a_arg, gm_a):
-            return False
+    for node_a in nodes:
```

Comment:
is it possible to have a function to return `nodes[num:]`?

Reply:
yeah, I looked into it, didn't see a way that was significantly better than the current code